VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
The reason is because there are .appx files created outside the AppPackages/ folder, in the project root folder. And this rule is for this scenario. 

I attached a screenshot from GitHub Desktop.

![2017-05-09](https://cloud.githubusercontent.com/assets/20776298/25983248/29ddb92e-36ec-11e7-91d0-e4f1ae12120f.png)